tests/test_user_stream.py

```diff
@@ -96,6 +96,23 @@ def test_receiving_order_cancelled(self):
     self.assertEqual(self.listener.order_cancelled, order_cancelled)
     self.assertEqual(self.listener.message, order_cancelled)
 
+  def test_receiving_all_orders_cancelled(self):
+    all_orders_cancelled = {'type': 'all_orders_cancelled'}
+    self.user_stream.on_message(self.serialize_to_trader([all_orders_cancelled]))
+
+    self.assertEqual(self.listener.error, None)
+    self.assertEqual(self.listener.all_orders_cancelled, all_orders_cancelled)
+    self.assertEqual(self.listener.message, all_orders_cancelled)
+
+  def test_receiving_cancel_all_orders_failed(self):
+    cancel_all_orders_failed = {'type': 'cancel_all_orders_failed',
```

Comment:
Dict does not match the coding style in this file.

Reply:
What's different? It's just like in the previous tests (i. e. test_receiving_order_cancelled), it just doesn't fit in one line.

Reply:
I wasn't clear, this way:
```
cancel_all_orders_failed = {
  'type': 'cancel_all_orders_failed',
  'cause': 'session_not_active',
}
```

Reply:
OK